twitter_ads/enum.py

```diff
@@ -195,3 +195,25 @@ def enum(**enums):
     PUBLISHED='PUBLISHED',
     SCHEDULED='SCHEDULED'
 )
+
+SEGMENTATION_TYPE = enum(
+    AGE='AGE',
+    APP_STORE_CATEGORY='APP_STORE_CATEGORY',
+    AUDIENCES='AUDIENCES',
+    CITIES='CITIES',
+    CONVERSATIONS='CONVERSATIONS',
+    CONVERSION_TAGS='CONVERSION_TAGS',
+    DEVICES='DEVICES',
+    EVENTS='EVENTS',
+    GENDER='GENDER',
+    INTERESTS='INTERESTS',
+    KEYWORDS='KEYWORDS',
+    LANGUAGES='LANGUAGES',
+    LOCATIONS='LOCATIONS',
+    PLATFORMS='PLATFORMS',
+    PLATFORM_VERSIONS='PLATFORM_VERSIONS',
+    POSTAL_CODES='POSTAL_CODES',
+    REGIONS='REGIONS',
+    SIMILAR_TO_FOLLOWERS_OF_USER='SIMILAR_TO_FOLLOWERS_OF_USER',
+    TV_SHOWS='TV_SHOWS',
```

Comment:
Can you remove the trailing comma here? `,`

Reply:
:+1:  removed